tasks/24_index-to-insert/index-to-insert.js

```diff
@@ -0,0 +1,9 @@
+export function getIndexToInsert(arr, num) {
+  arr.sort((a, b)=> a-b);
+  for (let i = 0; i<arr.length-1; i++) {
+    if (num > arr[i] && num <=arr[i+1]) {
+      return i+1;
+    } else if (num <= arr[0]) return 0;
+    else if (num >arr[arr.length -1]) return arr.length;
```

Comment:
Should be consistent using {} in if statement. I suggest to use it all the times. All previous files should be fixed too

Reply:
Also please simplify these conditions

Reply:
Hint: define variable equals to 0 outside of the loop and increment it until this condition `num > arr[i]` is met.

Reply:
Thanks to your hint I was able to optimize the code to work without the `else if` statements, using a while loop.